backend/src/test/java/de/bund/digitalservice/ris/norms/utils/EidConsistencyGuardianTest.java

```diff
@@ -217,6 +218,7 @@ void itCorrectsMissingEids() {
     assertThat(diff.hasDifferences()).isFalse();
   }
 
+  @Disabled("we don't use eids in proprietary data")
```

Comment:
could we still have a "complex" test that the eids are corrected, or otherwise just delete this test?